WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -723,7 +724,24 @@ extension NotificationsViewController {
         view.isUserInteractionEnabled = false
 
         DispatchQueue.main.async {
-            self.performSegue(withIdentifier: NotificationDetailsViewController.classNameWithoutNamespaces(), sender: note)
+            if FeatureFlag.notificationCommentDetails.enabled,
+               note.kind == .comment {
+                let notificationCommentDetailCoordinator = NotificationCommentDetailCoordinator(notification: note)
+
+                // For now, NotificationCommentDetailCoordinator only loads the Comment if it is cached.
+                // If the comment is not cached, fall back to showing the old comment view.
+                // This is temporary until NotificationCommentDetailCoordinator can fetch the comment from the endpoint.
+
+                if let commentDetailViewController = notificationCommentDetailCoordinator.viewController {
+                    commentDetailViewController.navigationItem.largeTitleDisplayMode = .never
+                    self.showDetailViewController(commentDetailViewController, sender: nil)
+                } else {
+                    // TODO: remove when NotificationCommentDetailCoordinator updated to fetch comment.
+                    self.performSegue(withIdentifier: NotificationDetailsViewController.classNameWithoutNamespaces(), sender: note)
+                }
```

Comment:
Nitpick, but as this method is already pretty long, could we extract these into a `presentCommentDetail(for: note)` method or something?